plugins/org.eclipse.papyrus.umllight.ui.simplification/src/org/eclipse/papyrus/umllight/ui/simplification/internal/Activator.java

```diff
@@ -35,13 +40,13 @@ public Activator() {
 
 	public void start(BundleContext context) throws Exception {
 		super.start(context);
-
 		plugin = this;
+		configureCreationMenus();
+		ContextConfigurator.disableContext(Context.UML);
```

Comment:
If this is going to disable the UML context, it should also _enable_ the UML Light context, even though the UML Light context is registered with default activation.  But only enable UML Light if UML is found to be enabled and needs to be disabled.

As an intrepid user, I turned off the UML Light context and turned on UML in the preferences to access the standard properties views for my own reasons.  Then, on the next start of my workbench, the **UML** tab in the properties view was absent from every selection because the UML context was disabled and the UML Light also was still disabled in the preferences.